lib/msal-core/samples/react-sample-app/src/AuthProvider.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from "react";
+import * as Msal from "msal";
+
+const loginRequest = {
+    scopes: ["openid", "profile", "User.Read"]
+};
+
+const tokenRequest = {
+    scopes: ["Mail.Read"]
+};
+
+const graphConfig = {
+    graphMeEndpoint: "https://graph.microsoft.com/v1.0/me",
+    graphMailEndpoint: "https://graph.microsoft.com/v1.0/me/messages"
+};
+
+const msalApp = new Msal.UserAgentApplication({
+    auth: {
+        clientId: "245e9392-c666-4d51-8f8a-bfd9e55b2456",
+        authority: "https://login.microsoftonline.com/common",
+        validateAuthority: true,
+        postLogoutRedirectUri: "http://localhost:3000"
+    },
+    cache: {
+        cacheLocation: "localStorage",
```

Comment:
The default setting is 'sessionStorage'. Can we stick to that? I know we are going off existing samples. Let us discuss this and decide what patterns we want to establish to the devs in our samples. @DarylThayil.

Reply:
Sounds good